lbuild/module.py

```diff
@@ -396,6 +397,12 @@ def __init__(self,
         self.options = {}
 
     @property
+    def _logger(self):
+        if self.__logger is None and self._fullname is not None:
+            self.__logger = logging.getLogger(self._fullname)
```

Comment:
This breaks the logger name convention, ie. `modm:platform:gpio` instead of `modm.platform.gpio`. I think that's ok though.